mover-syncthing/Dockerfile

```diff
@@ -0,0 +1,85 @@
+# Build the Syncthing image from Golang
+FROM golang:1.17 AS syncthing-builder
+
+# source directory
+RUN mkdir -p /workspace
+WORKDIR /workspace
+
+# latest release as of writing this
+ARG SYNCTHING_VERSION="v1.18.4"
+ARG SYNCTHING_GIT_HASH="951b0589522e13e14e8b40d33ea66de1e69c8b4e"
+
+# clone & cd
+RUN git clone \
+  --depth 1 \
+  --branch=${SYNCTHING_VERSION} \
+  https://github.com/syncthing/syncthing.git
+WORKDIR /workspace/syncthing
+
+# Make sure we have the correct Syncthing release
+RUN /bin/bash -c "[[ $(git rev-list -n 1 HEAD) == ${SYNCTHING_GIT_HASH} ]]"
+# Create the binary for syncthing 
+RUN go run build.go
+
+
+ENTRYPOINT [ "/workspace/syncthing/bin/syncthing" ]
+
+# Build final container 
+FROM registry.access.redhat.com/ubi8-minimal
+
+# RUN useradd -ms /bin/bash stuser
+# USER root
+
+RUN microdnf update -y && \
+    microdnf clean all && \
+    rm -rf /var/cache/yum
+
+EXPOSE 21027/udp 2000/tcp 22000 8384/tcp
+
+COPY --from=syncthing-builder \
+  /workspace/syncthing/bin/syncthing \
+  /usr/local/bin/syncthing
+
+
+# general image args
+ARG builddate_arg="(unknown)"
+ARG version_arg="(unknown)"
+ENV builddate="${builddate_arg}"
+ENV version="${version_arg}"
+
+# set labels 
+LABEL org.label-schema.build-date="${builddate}" \
+      org.label-schema.description="Syncthing-based data mover for VolSync" \
+      org.label-schema.license="AGPL v3" \
+      org.label-schema.name="volsync-mover-syncthing" \
+      org.label-schema.schema-version="1.0" \
+      org.label-schema.vcs-ref="${version}" \
+      org.label-schema.vcs-url="https://github.com/backube/volsync" \
+      org.label-schema.vendor="Backube" \
+      org.label-schema.version="${version}"
+###### syncthing-specific args ############
+
+# Path where the config volume will be mounted
+ENV SYNCTHING_CONFIG_DIR="/config"
+
+# Bind the GUI address to 0.0.0.0 so we can get external requests
+ENV STGUIADDRESS=0.0.0.0:8384
+
+# Variables that will be used to configure the data volume on startup
+# folder path 
+ENV SYNCTHING_DATA_DIR="/data"
+# folder id - must be the same on all synced nodes
+ENV SYNCTHING_DATA_ID="my-folder-id" 
+# folder label
+ENV SYNCTHING_DATA_LABEL="synced volume"
```

Comment:
Is there a reason to customize these or could we just hardcode them in the config file?